tensorflow/python/ops/string_ops.py

```diff
@@ -122,37 +123,19 @@ def string_format(template, inputs, placeholder="{}", summarize=3, name=None):
 
   Example:
     Formatting a single-tensor template:
-    ```python
-    sess = tf.compat.v1.Session()
-    with sess.as_default():
-        tensor = tf.range(10)
-        formatted = tf.strings.format("tensor: {}, suffix", tensor)
-        out = sess.run(formatted)
-        expected = "tensor: [0 1 2 ... 7 8 9], suffix"
 
-        assert(out.decode() == expected)
-    ```
+    >>> tensor = tf.range(10)
+    >>> tf.strings.format("tensor: {}, suffix", tensor)
+    <tf.Tensor: shape=(), dtype=string, numpy=b'tensor: [0 1 2 ... 7 8 9], suffix'>
 
     Formatting a multi-tensor template:
-    ```python
-    sess = tf.compat.v1.Session()
-    with sess.as_default():
-        tensor_one = tf.reshape(tf.range(100), [10, 10])
-        tensor_two = tf.range(10)
-        formatted = tf.strings.format("first: {}, second: {}, suffix",
-          (tensor_one, tensor_two))
-
-        out = sess.run(formatted)
-        expected = ("first: [[0 1 2 ... 7 8 9]\n"
-              " [10 11 12 ... 17 18 19]\n"
-              " [20 21 22 ... 27 28 29]\n"
-              " ...\n"
-              " [70 71 72 ... 77 78 79]\n"
-              " [80 81 82 ... 87 88 89]\n"
-              " [90 91 92 ... 97 98 99]], second: [0 1 2 ... 7 8 9], suffix")
-
-        assert(out.decode() == expected)
-    ```
+
+    >>> tensor_one = tf.reshape(tf.range(100), [10, 10])
```

Comment:
Let's try a shorter range so that we don't need to have a line too long and disable pylint